Collector_2019-08-05.sql

```diff
@@ -0,0 +1,57 @@
+# ************************************************************
```

Comment:
Some VARCHARs could potentially be replaced with ENUMs to restrict the amount options available for things like position and soil type

Adding a separate field for the latin name of plants would make things much easier when you get to displaying this information on the front end